guides/common/modules/proc_configuring-pull-based-transport-for-remote-execution.adoc

```diff
@@ -0,0 +1,17 @@
+[id="configuring-pull-based-transport-for-remote-execution_{context}"]
+= Configuring Pull-based Transport for Remote Execution
+
+Use this procedure to set up `pull-mqtt` mode for pull-based remote execution transport.
+
+.Procedure
+ifdef::foreman[]
+. Ensure your machine certificates are installed.
+** If you use Puppet, you can use Puppet certificates.
+. Install `yggdrasil` and `foreman_ygg_worker`.
+. Configure `yggdrasil`.
+. Start `yggdrasil`.
+endif::[]
+ifdef::katello[]
+. Retrieve the `katello-pull-transport-migrate` script for {Project} from your client repository.
```

Comment:
```suggestion
. Install the `katello-pull-transport-migrate` package from the {Team} {project-client-name} repository.
```

Installing the RPM runs the script for you, so running the script explicitly is not needed.